recipes/piscem/build_rs.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/build.rs b/build.rs
+index 9a5b4c1..7aa001f 100644
+--- a/build.rs
++++ b/build.rs
+@@ -67,6 +67,16 @@ fn main() {
+         dst_piscem_cpp.join("lib").display()
+     );
+
++    // Arghhh conda!
```

Comment:
Is this still needed?

Reply:
The comment or the patch? The patch is necessary, that’s what allows it to build on conda. The comment is because across Linux and osx systems, this “fix” is only required to build in the conda environment. Please feel free to change the comment itself, but something providing context that this is a conda specific fix, and not in general required, seems informative.

Reply:
I wonder what in the Conda build environment removes these folders from the default library path ?!

Reply:
I rather think it’s that everywhere else these are placed in lib but in the conda build system (on Linux only) lib64 instead. I don’t know why though. Regardless, this fixes it because the paths in build.rs are aggregation.